example.cpp

```diff
@@ -168,7 +170,7 @@ TEST_CASE_METHOD(CoherentUnitTestFixture, "test_xor")
     WARN(qftReg);
 }
 
-TEST_CASE_METHOD(CoherentUnitTestFixture, "test_add")
+/*TEST_CASE_METHOD(CoherentUnitTestFixture, "test_add")
```

Comment:
Did you mean to comment out all of these tests?

Reply:
We need tests, but they'll take a different form, so they've been removed.